services/galley/src/Galley/API/Teams/Features.hs

```diff
@@ -336,7 +336,7 @@ getSSOStatusInternal ::
 getSSOStatusInternal =
   either
     (const $ Public.TeamFeatureStatusNoConfig <$> getDef)
-    (getFeatureStatusNoConfig @'Public.WithoutLockStatus @'Public.TeamFeatureSSO getDef)
+    (getFeatureStatusNoConfig @'Public.TeamFeatureSSO getDef)
```

Comment:
I don't understand this change.  The internal end-point has to return the lock status, the external shouldn't.  Both handlers call this function, so this function has to take an `IncludeLockStatus` as argument and then pass it along to `getFeatureStatusNoConfig`.

It's analogous to the with-config case.